ui/lib/ldap/addon/components/page/configuration.ts

```diff
@@ -0,0 +1,82 @@
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+import { task } from 'ember-concurrency';
+import { waitFor } from '@ember/test-waiters';
+import errorMessage from 'vault/utils/error-message';
+
+import type LdapConfigModel from 'vault/models/ldap/config';
+import type SecretEngineModel from 'vault/models/secret-engine';
+import type AdapterError from 'ember-data/adapter'; // eslint-disable-line ember/use-ember-data-rfc-395-imports
+import type { Breadcrumb } from 'vault/vault/app-types';
+import type FlashMessageService from 'vault/services/flash-messages';
+
+interface Args {
+  configModel: LdapConfigModel;
+  configError: AdapterError;
+  backendModel: SecretEngineModel;
+  breadcrumbs: Array<Breadcrumb>;
+}
+
+interface Field {
+  label: string;
+  value: any; // eslint-disable-line @typescript-eslint/no-explicit-any
+  formatTtl?: boolean;
+}
+
+export default class LdapConfigurationPageComponent extends Component<Args> {
+  @service declare readonly flashMessages: FlashMessageService;
+
+  get defaultFields(): Array<Field> {
+    const model = this.args.configModel;
+    const keys = [
+      'binddn',
+      'url',
+      'schema',
+      'password_policy',
+      'userdn',
+      'userattr',
+      'connection_timeout',
+      'request_timeout',
+    ];
+    return model.allFields.reduce<Array<Field>>((filtered, field) => {
+      if (keys.includes(field.name)) {
+        const label =
+          {
+            schema: 'Schema',
+            password_policy: 'Password Policy',
+          }[field.name] || field.options.label;
```

Comment:
You could instead use the [toLabel](https://github.com/hashicorp/vault/blob/e55c18ed1299e0d36b88e603fa9f12adaf8e75dc/ui/app/helpers/to-label.js) helper, might make this read a little cleaner
`const label = field.options.label || toLabel(field.name)`

Reply:
Amazing! I didn't know this existed, thanks.

Reply:
Darn it's in the main app so I think I will opt to leave as is for now and make up a ticket to move it to the core addon for wider use.

Reply:
It will need some updating to follow our new capitalization format I think.